cmd/reviewdog/main.go

```diff
@@ -317,23 +317,67 @@ func githubService(ctx context.Context) (gs *githubservice.GitHubPullRequest, is
 	if err != nil {
 		return nil, isPR, err
 	}
-	// TODO: support commit build
-	if !isPR {
-		return nil, isPR, nil
-	}
 
 	client, err := githubClient(ctx, token)
 	if err != nil {
 		return nil, isPR, err
 	}
 
+	if !isPR {
+		pullRequests, _, err := client.PullRequests.List(ctx, g.Owner, g.Repo, &github.PullRequestListOptions{})
```

Comment:
Instead of list pull-request with no conditions, use PullRequestListOptions to specify head SHA and branch respectively.
Also, can you explicitly add other conditions (status, sort, direction)? 
https://github.com/google/go-github/blob/4e1a504644bfd34564773b2f8c4c2ba91c8e21ec/github/pulls.go#L110

The reason is that there may be more than N open pull requests in the repository and cannot fetch the target p-r unless we uses the above condition and/or support pagination.